Formula/clickhouse-cpp.rb

```diff
@@ -0,0 +1,77 @@
+class ClickhouseCpp < Formula
+  desc "C++ client library for ClickHouse"
+  homepage "https://github.com/ClickHouse/clickhouse-cpp#readme"
+  url "https://github.com/ClickHouse/clickhouse-cpp.git",
+      tag:      "1.5.0",
+      revision: "1415b5936a2ac2f084850b09057e05fb5798b2f1"
+  license "Apache-2.0"
+  head "https://github.com/ClickHouse/clickhouse-cpp.git", branch: "master"
+
+  depends_on "cmake" => [:build, :test]
```

Comment:
Looks like you're missing an `abseil` dependency, based on the test output.

Reply:
Looks like yes. However, It should contain the relevant parts of `abseil` in its source tree. Does this fail only in `arm64`? Aren't the set of installed packages the same in all environments?

Reply:
They should be. The test fails on Intel macOS too from a missing `abseil`.

Reply:
Added.